#!/usr/bin/python3
def raise_exception_msg(message=""):
    try:
        almah = boy
        print(alma)
    except NameError:
        print('NameError occured, some variable not defined')
